api/types/server_info.go

```diff
@@ -166,6 +169,9 @@ func (s *ServerInfoV1) SetNewLabels(labels map[string]string) {
 func (s *ServerInfoV1) setStaticFields() {
 	s.Kind = KindServerInfo
 	s.Version = V1
+	if s.SubKind == "" {
```

Comment:
What other valid SubKinds do we expect here? I'm curious why not just always set it to SubKindCloudInfo.